.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug Report
+about: Report a bug in the framework or a POD
+labels: Bug
+---
+
```

Comment:
This is great @wrongkindofdoctor - what are your thoughts on having the submitter describe the criticality of the bug or some other measure of the impact?

Reply:
I can add a line to indicate the bug severity--maybe rank from 1--3, where 1 is something like a minor issue that affects one POD and 3 is something like a certain supported Python version causing the framework to crash.